include/zephyr/drivers/dma.h

```diff
@@ -582,6 +595,30 @@ static inline int dma_get_status(const struct device *dev, uint32_t channel,
 	return -ENOSYS;
 }
 
+/**
+ * @brief get attribute of a dma controller
+ *
+ * Implementations must check the validity of the type passed in and
```

Comment:
Be more verbose pls, the comment gives no clue what the procedure really does - what "an attribute", etc.???

"a device specific static or runtime attribute like required alignment size" or similar